ci-operator/step-registry/ipi/install/install/in-bastion/ipi-install-install-in-bastion-commands.sh

```diff
@@ -27,7 +27,7 @@ function run_ssh_cmd() {
     local host=$3
     local remote_cmd=$4
 
-    options=" -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null "
+    options=" -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ServerAliveInterval=300 -o ServerAliveCountMax=10 "
```

Comment:
lgtm

Reply:
one more question about this step:
For GCP, we don't need to upload any credential file into bastion if bastion already auth with sa, right? If so, I guess message [here](https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/in-bastion/ipi-install-install-in-bastion-commands.sh#L149) will be printed out, it might be confused. How about to update this echo message to be more accurate? 
something like this,
`echo "Unsupported cluster type '${CLUSTER_TYPE}'" `
=> `echo "No need to upload any credential files into bastion host for cluster type '${CLUSTER_TYPE}'"`